lib/js/icons/fontawesome.ts

```diff
@@ -113,6 +114,7 @@ import { faPaperPlane } from '@fortawesome/pro-regular-svg-icons/faPaperPlane';
 import { faPencil } from '@fortawesome/pro-regular-svg-icons/faPencil';
 import { faPenCircle } from '@fortawesome/pro-regular-svg-icons/faPenCircle';
 import { faPenField } from '@fortawesome/pro-regular-svg-icons/faPenField';
+import { faPills } from '@fortawesome/pro-solid-svg-icons/faPills';
```

Comment:
we mark solid icons with `fas` prefix 
```
import { faPills as fasPills } from '@fortawesome/pro-solid-svg-icons/faPills';
```

Also you could move the import down to other solid icon imports